lib/orogen/gen/tasks.rb

```diff
@@ -517,12 +517,16 @@ def generate
                 if(has_dynamic_attributes?)
                     create_dynamic_updater("updateDynamicAttributes",superclass.has_dynamic_attributes?)
                 end
-                
-                
+
+
                 extensions.each do |ext|
                     if ext.respond_to?(:early_register_for_generation)
+                        OroGen.warn "The plugin #{ext.name} defines a \"early_register_for_generation\" hook, this got renamed to \"pre_generation_hook\" please adapt the code or contact the developer."
                         ext.early_register_for_generation(self)
                     end
+                    if ext.respond_to?(:pre_generation_hook)
```

Comment:
I really don't see the point of renaming this. Both names are valid, and tell the same thing.